### PR TITLE
Docker: add portablecl to base image and remove cl image

### PR DESCRIPTION
**Pull request by jnewb1**
this allows me to run modeld in the container (on the CPU), although the modeld performance is pretty poor (3fps)

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
What's 3FPS? MetaDrive, modeld, or something else?

**Comment by jnewb1** [comment]
> What's 3FPS? MetaDrive, modeld, or something else?

modeld

**Comment by jnewb1** [comment]
POCL seems to work fine in the base image. model tests take about 40% longer, but still well under 10 minutes in CI

File: tools/sim/build_container.sh
Comment (jnewb1) [inline tools/sim/build_container.sh:1]: should just use selfdrive/test/docker_build.sh sim

**Comment by adeebshihadeh** [comment]
Are there any extras we can turn off? That's a big hit.

**Comment by jnewb1** [comment]
found that there's a prebuilt package and with that, its only 500mb larger

**Comment by adeebshihadeh** [comment]
If that's as small as we can get, I think that's an acceptable tradeoff. Remaining question is whether it's fast enough to use for the sim on a local machine.

File: Dockerfile.openpilot_base [hunk 1]
```diff
@@ -1,3 +1,7 @@
+FROM condaforge/mambaforge as pocl-builder
+
+RUN mamba install pocl
```
Comment (jnewb1) [inline Dockerfile.openpilot_base:3]: this conda POCL package has opencl drivers for nvidia cuda, intel, and amd

Comment (adeebshihadeh) [reply Dockerfile.openpilot_base:3]: how big is it?

**Pull request closed without merging**